etc/profile-a-l/liferea.profile

```diff
@@ -49,7 +50,10 @@ shell none
 tracelog
 
 disable-mnt
+#private-bin liferea
+private-cache
 private-dev
+private-etc alternatives,ca-certificates,crypto-policies,dconf,fonts,gtk-2.0,gtk-3.0,hostname,hosts,mime.types,nsswitch.conf,pki,pulse,resolv.conf,ssl,X11,xdg
```

Comment:
On Arch Linux with a user ID of 1001 I get:
```
Error: cannot read UID_MIN and/or GID_MIN from /etc/login.defs, using 1000 by default
```
Please add `login.defs` to private-etc.